frame/support/test/Cargo.toml

```diff
@@ -14,23 +14,26 @@ targets = ["x86_64-unknown-linux-gnu"]
 [dependencies]
 serde = { version = "1.0.101", default-features = false, features = ["derive"] }
 codec = { package = "parity-scale-codec", version = "2.0.0", default-features = false, features = ["derive"] }
+scale-info = { version = "0.9.0", default-features = false, features = ["derive"] }
+sp-arithmetic = { version = "3.0.0", default-features = false, path = "../../../primitives/arithmetic" }
 sp-io = { version = "3.0.0", path = "../../../primitives/io", default-features = false }
 sp-state-machine = { version = "0.9.0", optional = true, path = "../../../primitives/state-machine" }
 frame-support = { version = "3.0.0", default-features = false, path = "../" }
 sp-runtime = { version = "3.0.0", default-features = false, path = "../../../primitives/runtime" }
 sp-core = { version = "3.0.0", default-features = false, path = "../../../primitives/core" }
 sp-std = { version = "3.0.0", default-features = false, path = "../../../primitives/std" }
+sp-version = { version = "3.0.0", default-features = false, path = "../../../primitives/version" }
```

Comment:
Where does `sp-version` and `sp-arithmetic` get used in the tests?

Reply:
https://github.com/paritytech/substrate/blob/5de104ed5743307c2ee56f901f0cb972c17905d8/frame/support/test/tests/pallet_instance.rs#L724

https://github.com/paritytech/substrate/blob/5de104ed5743307c2ee56f901f0cb972c17905d8/frame/support/test/tests/pallet_instance.rs#L606

That said, I need to see whether it is feasible to do these tests without having to reproduce the metadata for the `System` pallet.

Reply:
I think it is ok to add those dependencies, and testing for system metadata again here